src/test/java/test/listeners/ordering/ListenerInvocationDefaultBehaviorTest.java

```diff
@@ -293,12 +293,12 @@ public void testOrderHasPassedFailedSkippedMethods() {
         IINVOKEDMETHODLISTENER_BEFORE_INVOCATION,
         IINVOKEDMETHODLISTENER_AFTER_INVOCATION,
         ITESTLISTENER_ON_TEST_FAILURE_TEST_METHOD,
+        IEXECUTION_VISUALISER_CONSUME_DOT_DEFINITION,
```

Comment:
The change to this test is because, as far as I can tell, this test was testing for undocumented behavior. The call to `consumeDotDefinition` is still happening after the test method has finished running, but now it's happening right after the method finishes running, rather than being "batched" at the end. I looked all over for documentation on how this was supposed to work, but all I could find was the Javadoc on the class itself: https://github.com/cbeust/testng/blob/master/src/main/java/org/testng/IExecutionVisualiser.java#L3

If this is something that absolutely can't change, please let me know and I can look for a workaround. Again, from everything I can tell, this was undocumented behavior that no one should be relying on, and in my Google searching, I didn't find any external references to IExecutionVisualiser.